testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.nullable_03/nullable.string.jst

```diff
@@ -0,0 +1,33 @@
+JSIGHT 0.3
```

Comment:
I put the comment here, but it is not related to "nullable.string" tests only. I'd like you to add another test for explicit `nullable: false` rule with all the types. You can add additional file, or add `nullable: false` test to each of the existing "nullable" files.

Reply:
fixed https://github.com/jsightapi/jsight-api-core/pull/15/commits/427fa5283974abfb514644608f0692f719a51663